src/cBiff12Writer.cls

```diff
@@ -130,6 +136,21 @@ Public Function Init( _
     Init = True
 End Function
 
+Public Function AddSheet(Optional ByVal SheetName As String, Optional ByVal lColCount As Long) As Boolean
+    pvEndSheet m_oSheetPart
+    Set m_oSheetPart = m_oContainer.SheetPart(m_oContainer.SheetPartCount + 1)
+    m_oSheetPart.SheetName = IIf(Not IsMissing(SheetName), SheetName, "Sheet" & Format(m_oContainer.SheetPartCount, "0"))
+    '--- reset previous variables
+    m_lRowCount = 0
+    m_lColCount = Iif(Not IsMissing(lColCount), lColCount, 1)
```

Comment:
Using `IsMissing` for `Long` optionals is not possible.

Can we just keep current `m_lColCount` when `lColCount` is not supplied with a simple `If` statement?